tests/unittest/llmapi/test_llm_utils.py

```diff
@@ -41,7 +40,7 @@ def test_CachedModelLoader():
     args = LlmArgs(model=llama_model_path,
                    kv_cache_config=KvCacheConfig(free_gpu_memory_fraction=0.4))
     args.enable_build_cache = True
-    args._setup()
+    args.model_post_init({})
```

Comment:
why do you need to call this explicitly? The Pydantic model should call this automatically during init

Reply:
Here the member is assigned after the instance creation. Will the validator be invoked automatically after that?

Reply:
Why can't you set the argument during initialization? 

and yes you are right. field/model validators won't be called when you update a field value after init